---a/net/minecraft/entity/item/EntityMinecartEmpty.java
+++b/net/minecraft/entity/item/EntityMinecartEmpty.java
@@ -4,31 +4,25 @@
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.world.World;
 
-public class EntityMinecartEmpty extends EntityMinecart
-{
+public class EntityMinecartEmpty extends EntityMinecart {
+
     private static final String __OBFID = "CL_00001677";
 
-    public EntityMinecartEmpty(World worldIn)
-    {
+    public EntityMinecartEmpty(World worldIn) {
         super(worldIn);
     }
 
-    public EntityMinecartEmpty(World worldIn, double p_i1723_2_, double p_i1723_4_, double p_i1723_6_)
-    {
+    public EntityMinecartEmpty(World worldIn, double p_i1723_2_, double p_i1723_4_, double p_i1723_6_) {
         super(worldIn, p_i1723_2_, p_i1723_4_, p_i1723_6_);
     }
 
-    public boolean interactFirst(EntityPlayer playerIn)
-    {
-        if (this.riddenByEntity != null && this.riddenByEntity instanceof EntityPlayer && this.riddenByEntity != playerIn)
-        {
+    public boolean interactFirst(EntityPlayer playerIn) {
+        if (this.riddenByEntity != null && this.riddenByEntity instanceof EntityPlayer && this.riddenByEntity != playerIn) {
             return true;
-        }
-        else if (this.riddenByEntity != null && this.riddenByEntity != playerIn)
-        {
+        } else if (this.riddenByEntity != null && this.riddenByEntity != playerIn) {
             return false;
-        }
-        else
-        {
-            if (!this.worldObj.isRemote)
-            {
+        } else {
+            if (!this.worldObj.isRemote) {
                 playerIn.mountEntity(this);
             }
 
@@ -39,22 +39,16 @@
         }
     }
 
-    public void onActivatorRailPass(int p_96095_1_, int p_96095_2_, int p_96095_3_, boolean p_96095_4_)
-    {
-        if (p_96095_4_)
-        {
-            if (this.riddenByEntity != null)
-            {
-                this.riddenByEntity.mountEntity((Entity)null);
+    public void onActivatorRailPass(int p_96095_1_, int p_96095_2_, int p_96095_3_, boolean p_96095_4_) {
+        if (p_96095_4_) {
+            if (this.riddenByEntity != null) {
+                this.riddenByEntity.mountEntity((Entity) null);
             }
 
-            if (this.getRollingAmplitude() == 0)
-            {
+            if (this.getRollingAmplitude() == 0) {
                 this.setRollingDirection(-this.getRollingDirection());
                 this.setRollingAmplitude(10);
                 this.setDamage(50.0F);
@@ -58,9 +58,7 @@
         }
     }
 
-    public EntityMinecart.EnumMinecartType getMinecartType()
-    {
+    public EntityMinecart.EnumMinecartType getMinecartType() {
         return EntityMinecart.EnumMinecartType.RIDEABLE;
     }
-}
+}
